source/manual/sidekiq.html.md

```diff
@@ -45,6 +45,12 @@ Or on AWS:
 $ ssh $(ssh integration "govuk_node_list --single-node -c backend").integration -CNL 9000:127.0.0.1:3211
 ```
 
+or using `govuk-connect`:
+
+```bash
+$ govuk-connect sidekiq-monitoring -e production aws/backend 
```

Comment:
I don't think we should do this until govuk-connect is [official](https://github.com/alphagov/govuk-aws/issues/1159). It'll be soon but not yet. And we should do the things mentioned in the comments, like using `gds govuk connect` so there's one tool. @benthorner We should expedite this.

Reply:
I thought it would useful to have _something_, so we're not stuck like we were last night. If there's some other way that works, I'm happy to document that.

Reply:
You know what yeah let's do this.

```suggestion
$ brew install alphagov/gds/govuk-connect
$ gds govuk connect sidekiq-monitoring -e production aws/backend 
```